File: Código/Loja/src/dados/Cliente.java
package dados;

public class Cliente {
    private int id;
    private String nome;
    private String cpf;
    private String rg;
    private int idade;

    public Cliente() {

    }

    public Cliente(String nome, String cpf, String rg) {
        this.nome = nome;
        this.cpf = cpf;
        this.rg = rg;
    }

    public Cliente(int id, String nome, String cpf, String rg) {
        this.id = id;
        this.nome = nome;
        this.cpf = cpf;
        this.rg = rg;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getIdade() {
        return idade;
    }

    public void setIdade(int idade) {
        this.idade = idade;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getCpf() {
        return cpf;
    }

    public void setCpf(String cpf) {
        this.cpf = cpf;
    }

    public String getRg() {
        return rg;
    }

    public void setRg(String rg) {
        this.rg = rg;
    }

    @Override
    public String toString() {
        return "Cliente [nome=" + nome + ", cpf=" + cpf + ", rg=" + rg + ", idade=" + idade + "]";
    }

}

File: Código/Loja/src/dados/Cliente.py
from dados.Pessoa import Pessoa
from dados.Carrinho import Carrinho


class Cliente(Pessoa):
    def __init__(self, nome, cpf, rg, telefone, endereco, email):
        super().__init__(nome, cpf, rg, telefone, endereco, email)
        self.isCadastrado = False
        self.carrinho = Carrinho()

    def get_is_cadastrado(self):
        return self.isCadastrado

    def set_is_cadastrado(self, isCadastrado):
        self.isCadastrado = isCadastrado

    def get_carrinho(self):
        return self.carrinho

    def __str__(self):
        return f"Cliente [nome={self.get_nome()}, cpf={self.get_cpf()}, rg={self.get_rg()}, endereco={self.get_endereco()}, email={self.get_email()}, telefone={self.get_telefone()}, isCadastrado={self.get_is_cadastrado()}]"
